concepts/observability-strategy/configurable-logging/08-logging-revised-api.md

```diff
@@ -275,16 +275,4 @@ spec:
       Labels:
         "job": "telemetry-fluent-bit"
       RemoveKeys: ["kubernetes", "stream"]
```

Comment:
Also make the other attributes lower-case

Reply:
fixed